src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -96,7 +96,9 @@
                                     OR
                                     sample_unique_id NOT IN (
                                         <foreach item="sampleIdentifier" collection="studyViewFilterHelper.customDataSamples" separator=",">
-                                            '${sampleIdentifier.studyId}_${sampleIdentifier.sampleId}'
+                                            <if test="sampleIdentifier.getUniqueSampleId != null">
+                                                #{sampleIdentifier.getUniqueSampleId}
+                                            </if>
```

Comment:
I'm not sure if adding `test` at here will break anything. maybe we need to add another `'',` at here like we did in line 85. Maybe @gblaih can comment on this.

Reply:
there is a case if getUniqueSampleId is null for all customDataSamples, it will break since we cannot do NOT IN (). adding '' will prevent it from breaking, but it may cause unintended issues with the logic i think. is there a case where the custom samples have null studyIds and sampleIds?